modules/javafx.controls/src/main/java/javafx/scene/control/skin/VirtualFlow.java

```diff
@@ -2485,13 +2556,22 @@ private void releaseCell(T cell) {
         }
     }
 
+
     /**
-     * This method is an experts-only method - if the requested index is not
-     * already an existing visible cell, it will create a cell for the
-     * given index and insert it into the sheet. From that point on it will be
-     * unmanaged, and is up to the caller of this method to manage it.
+     * Creates and returns a new cell for the given index.
+     * <p>
+     * If the requested index is not already an existing visible cell, it will create a cell for the given index and
+     * insert it into the sheet. If the index is existing, simply returns the visible cell. From that point on it will
+     * be unmanaged, and is up to the caller of this method to manage it.
+     * <p>
+     * This is useful if you need a row that should not be visible (a row that always stick to the top for example). It
+     * allows you to easily creates it and be sure it's correctly initialized and inserted in the {@code VirtualFlow}.
+     *
```

Comment:
Avoid "you": "useful in the case of a row", or "useful if one needs a row", or "useful if a row... is desired" or something similar.